apps/avatax/src/modules/avatax/order-confirmed/avatax-order-confirmed-payload-transformer.test.ts

```diff
@@ -32,14 +32,19 @@ const transformer = new AvataxOrderConfirmedPayloadTransformer(
 
 describe("AvataxOrderConfirmedPayloadTransformer", () => {
   it("returns document type of SalesInvoice when isDocumentRecordingEnabled is true", async () => {
-    const payload = await transformer.transform(orderMock, saleorOrderMock, avataxConfigMock, []);
+    const payload = await transformer.transform(
+      orderMock,
+      saleorOrderConfirmedEvent,
+      avataxConfigMock,
+      [],
```

Comment:
prefer to use object attribute. once reading code I have no idea what is `[]` here. using object spread makes it obvious